supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -94,7 +96,9 @@ class JesInitializationActor(override val workflowDescriptor: BackendWorkflowDes
       val upload = () => Future(path.writeAsJson(content))
 
       workflowLogger.info(s"Creating authentication file for workflow ${workflowDescriptor.id} at \n ${path.toString}")
-      Retry.withRetry(upload, isFatal = isFatalJesException, isTransient = isTransientJesException)(context.system) map { _ => () }
+      Retry.withRetry(upload, isFatal = isFatalJesException, isTransient = isTransientJesException)(context.system) map { _ => () } recover {
+        case failure => throw new IOException("Failed to upload authentication file", failure)
```

Comment:
Yes it's equivalent, instead of throwing the exception you need to wrap it in a `Future` though, I usually don't use recoverWith unless the recovery work actually needs to be asynchronous but I honestly don't care if people prefer this way I'll change it, this is mainly so I can add a failure check on the centaur test.